nfcore_custom.config

```diff
@@ -47,7 +48,9 @@ profiles {
 params {
   // This is a groovy map, not a nextflow parameter set
   hostnames = [
+    binac: ['.binac.uni-tuebingen.de'],
```

Comment:
Are these valid now? They have not been when I checked last time? Same below for CFC ...?

Reply:
That's contained in the Hostnames when I log in

Reply:
doesn't it work like this?

Reply:
It does :-) Then they added that apparently :-)

Reply:
Maybe check if nodes have that too 😓

Reply:
I don't think that it matters about the nodes here.. As I remember, this `hostnames` array is only used to warn the user if they are not using the relevant config profile when they launch Nextflow (and maybe again if it exits with an error).

Reply:
ok, then it should be fine, thanks, I'll edit it again if it does not work :)